package exception;

public class ExeptionExample {

	public static void main(String[] args) {
		// TODO 自動生成されたメソッド・スタブ

		int a = 4;
		int b = 0;
		int c = a / b;
		System.out.println("cの値は" + c);
		System.out.println("処理を正常に終了します");
	}

}
